### ISSUE TITLE
Focus lost/misplaced on date change in IE11

### ISSUE THREAD

**Issue by CrossTheDev** [airbnb/react-dates#2012]
**react-dates version**
react-dates@21.8.0

**Describe the bug**
This is an accessibility problem in IE11. Focus is lost if you select a date by mouse or keyboard from calendar. You should be able to TAB to the next input, but it is not possible in IE11.

**Steps to reproduce**
1. Open calendar
1. Select a date by mouse or keyboard
1. Use TAB to move to next input
1. You are now not in the next input but somewhere else on the page

**Is the issue reproducible in Storybook?**
http://airbnb.io/react-dates/?path=/story/singledatepicker-sdp--as-part-of-a-form
